### PR TITLE
PR for recipe of Jupyter enterprise gateway v7

### PR DESCRIPTION
**Pull request by liukun1016**
This is the PR for our open source project Jupyter Enterprise Gateway on jupyter-incubator: https://github.com/jupyter-incubator/enterprise_gateway

### CODE REVIEW

**Comment by xoviat** [comment]
You need to skip windows and OSX or fix the builds.

**Comment by liukun1016** [comment]
I got an exception for a python package:

```
conda.exceptions.ResolvePackageNotFound: 
  - yarn-api-client 0.2.3
```

Any idea about this exception? @xoviat

File: recipes/jupyter_enterprise_gateway/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (dougalsutherland) [inline recipes/jupyter_enterprise_gateway/meta.yaml]: Nit: you could delete this blank line.

File: recipes/jupyter_enterprise_gateway/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: 'A web server for spawning and communicating with remote Jupyter kernels'
+
+  description: |
+    Jupyter Enterprise Gateway is a lightweight, multi-tenant, scalable and secure gateway that enables
+    Jupyter Notebooks to share resources across an Apache Spark cluster
+  doc_url: http://jupyter-enterprise-gateway.readthedocs.io/en/latest/
+  dev_url: https://github.com/jupyter-incubator/jupyter_enterprise_gateway
```
Comment (dougalsutherland) [inline recipes/jupyter_enterprise_gateway/meta.yaml]: Wrong URL here too.

File: recipes/jupyter_enterprise_gateway/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+extra:
+  recipe-maintainers:
+    - liukun1016
+    - lresende
```
Comment (dougalsutherland) [inline recipes/jupyter_enterprise_gateway/meta.yaml]: @lresende, are you happy to be a maintainer for the conda-forge package of jupyter_enterprise_gateway?

File: recipes/jupyter_enterprise_gateway/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    - enterprise_gateway.services
+    - enterprise_gateway.services.kernels
+    - enterprise_gateway.services.kernelspecs
+    - enterprise_gateway.services.sessions
+  commands:
+    - jupyter-enterprisegateway --help
+    - jupyter enterprisegateway --help
+
+about:
+  home: https://github.com/jupyter-incubator/jupyter_enterprise_gateway
```
[the remark this answers is not in the document]
Comment (liukun1016) [reply recipes/jupyter_enterprise_gateway/meta.yaml]: My bad. Didn't know why I put jupyter_enterprise_gateway there.

**Comment by dougalsutherland** [comment]
Looks ready to merge other than confirmation from @lresende. If you want to take him off and merge now we can do that too; can always add in later once the feedstock is created.

**Comment by liukun1016** [comment]
@dougalsutherland Sounds good. Let me add him later so we could merge this PR now.

**Comment by dougalsutherland** [comment]
Done! Thanks @liukun1016.

**Comment by lresende** [comment]
Sorry for the delay responding, have been traveling... 
Feel free to add me as maintainer.

**Pull request merged**